### PR TITLE
[13.0][IMP] delivery_tnt_oca: Remove unnecessary BOOK tag from ACTIVITY

### PR DESCRIPTION
**Pull request by victoralmau**
Remove unnecessary `BOOK` tag from `ACTIVITY`

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT40767

### CODE REVIEW

File: delivery_tnt_oca/models/tnt_request.py [hunk 1]
```diff
@@ -291,6 +291,9 @@ def _prepare_create_shipping(self):
                 },
             },
         }
+        # This may be optional if the customer has fixed pick-up.
+        if self.activity_book:
+            data["ACTIVITY"]["BOOK"] = {"CONREF": self.record.name}
```
Comment (pedrobaeza) [inline delivery_tnt_oca/models/tnt_request.py]: What is `self.record.name`? There's no options on this?

Comment (victoralmau) [reply delivery_tnt_oca/models/tnt_request.py]: `self.record.name` = `picking.name`

According to tnt support, in some specific cases it is not necessary to send the BOOK key.

Comment (pedrobaeza) [reply delivery_tnt_oca/models/tnt_request.py]: But explain what is `BOOK` key. `tnt_activity_book` field name is not saying anything about its purpose.

Comment (victoralmau) [reply delivery_tnt_oca/models/tnt_request.py]: `Finally, BOOK` tag is not necessary.

**Review by pedrobaeza — APPROVED** [review]

**Comment by victoralmau** [comment]
Ping @chienandalu

**Review by chienandalu — APPROVED** [review]

**Pull request merged**